codice/ddf
# [2.19.x] G-4107 Add plugin for user model

#### What does this PR do?
Adds a plugin so that user model behavior can be modified downstream. 

#### Who is reviewing it? 

@cassandrabailey293 
@andrewzimmer 
@hayleynorton 
@abel-connexta 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@lambeaux

#### How should this be tested?
Build, profile:install
Open a workspace and view the 2D map. 
Verify the default map layer shows up
Verify that in the 'layers' dropdown on the top right of the map you see the default layer
Verify you can adjust the opacity of the layer, hide it completely, show it again, etc. 
Verify the same in the 3D Map
Check the dev console for any new errors that seem related to the map/map layer/user model.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: G-4107

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
